tty/uart16550/uart16550.c

```diff
@@ -78,7 +78,7 @@ static void set_baudrate(void *_uart, speed_t baud)
 	uart_t *uart = (uart_t *)_uart;
 	int baud_rate = libtty_baudrate_to_int(baud);
 
-	if (baud_rate <= 0 || baud_rate > 115200) {
+	if (baud_rate <= 0 || baud_rate > 460800) {
```

Comment:
I agree with @niewim19 - the ceiling condition can be omitted and then `baud_rate` (after division) checked if it fits `uint16_t`.